src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -274,9 +274,7 @@ public static CompositeConfigurationSource CreateDefaultConfigurationSource()
 
         internal bool IsIntegrationEnabled(string name)
         {
-            return TraceEnabled &&
-                   Integrations[name].Enabled != false &&
-                   !DisabledIntegrationNames.Contains(name);
+            return TraceEnabled && !IntegrationIsDisabled(name);
```

Comment:
Thanks for making the change I suggested, but I'm still not convinced of the result. Now we have two methods with names that _sound_ like they are opposite (`IsIntegrationEnabled` and `IntegrationIsDisabled`), but  they are not exactly (one checks `TraceEnabled` and the other one does not). Maybe we can just refactor the original code instead of adding the new method?

Here are two suggestions:
```csharp
internal bool IsIntegrationEnabled(string name)
{
	return TraceEnabled &&
		   (Integrations[name].Enabled == null || Integrations[name].Enabled == true) &&
		   !DisabledIntegrationNames.Contains(name);
}
```

```csharp
internal bool IsIntegrationEnabled(string name)
{
    bool disabled = Integrations[name].Enabled == false || DisabledIntegrationNames.Contains(name);
    return TraceEnabled && !disabled;
}
```

Reply:
I see your point, it is confusing. I like the second one.